x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -61,6 +61,7 @@ xpack:
       realms:
         pki:
           pki1:
+            order: 1
```

Comment:
This page also has the following:

> If you are configuring multiple realms, you should explicitly set the order attribute.

... which I think should be changed to "you must explicitly set the order attribute..."

Reply:
Thanks. It's updated now. I also searched through the documentations to make sure there is no similar situation like this anymore.

Reply:
Here are the similar changes that are made based on above search result

Reply:
https://github.com/elastic/elasticsearch/pull/51195/commits/994c0af8d053c683eaca306770beaf3207c16dc5